README.md

```diff
@@ -0,0 +1,196 @@
+# curriculum-databases-projects-template
+
+> This template should be used for database related projects at Microverse.
+> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
```

Comment:
- Remove this guidelines as suggested. Add content relevant to the section instead. Do the same to all sections.